### PR TITLE
sys-apps/agedu: add 20211129

### PR DESCRIPTION
**Pull request by thesamesam**
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: sys-apps/agedu/agedu-20211129.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+BDEPEND="doc? ( app-doc/halibut )"
+
+# Notes:
```
Comment (thesamesam) [inline sys-apps/agedu/agedu-20211129.ebuild:24]: drop this once you're happy, just put it in here to explain a bit

**Comment by blueness** [comment]
This looks good!  I was stuck on how to turn off halibut _elegantly_.  Looks like a sledge hammer is the answer.

**Comment by thesamesam** [comment]
> This looks good! I was stuck on how to turn off halibut _elegantly_. Looks like a sledge hammer is the answer.

Oh, there's another option: -DCMAKE_DISABLE_FIND_PACKAGE_HALIBUT=$(usex !doc) or $(cmake_use_find_package halibut doc) too. I forgot about that!

**Comment by thesamesam** [comment]
Done in https://github.com/gentoo/gentoo/commit/1863ce1369d5845680284320391d3e3b633883b6.

**Pull request closed without merging**